### PR TITLE
Drums

### PR DESCRIPTION
**Pull request by RebeccaStankus**
Test Plan
https://highfidelity.testrail.net/index.php?/cases/view/12740
https://highfidelity.testrail.net/index.php?/cases/view/12743

### CODE REVIEW

File: Marketplace/drumKit/crash.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+//
+
+(function() {
+    var _this;
+
+    var AUDIO_VOLUME_LEVEL = 1;
+    var playing = false;
+    var sound;
+
+    var Cowbell = function() {
```
Comment (misslivirose) [inline Marketplace/drumKit/crash.js]: Rename to Crash

File: Marketplace/drumKit/hiHat.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    var AUDIO_VOLUME_LEVEL = 1;
+    var playing = false;
+    var sound;
+
+    var Cowbell = function() {
+        _this = this;
+    };
+
+    Cowbell.prototype = {
```
Comment (misslivirose) [inline Marketplace/drumKit/hiHat.js]: Rename to hiHat

File: Marketplace/drumKit/midTom.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            _this.injector = Audio.playSound(_this.sound, {position: _this.homePos, volume: AUDIO_VOLUME_LEVEL});
+            if (sound.downloaded && !playing) {
+                Audio.playSound(sound, {
+                    position: _this.homePos,
+                    volume: AUDIO_VOLUME_LEVEL
+                });
+                playing = true;
+                Script.setTimeout(function() {
+                    playing = false;
+                }, 250);
```
Comment (misslivirose) [inline Marketplace/drumKit/midTom.js]: Magic number

File: Marketplace/drumKit/ride.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+    var AUDIO_VOLUME_LEVEL = 0.8;
+    var HALF = 0.5;
+    var playing = false;
+    var soundEdge;
+    var soundBell;
+    var soundMid;
+    var topCenter;
+
+    var Cowbell = function() {
```
Comment (misslivirose) [inline Marketplace/drumKit/ride.js]: Needs less cowbell

File: Marketplace/drumKit/ride.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            _this.injector = Audio.playSound(_this.sound, {position: _this.homePos, volume: AUDIO_VOLUME_LEVEL});
+            if (specificSound.downloaded && !playing) {
+                Audio.playSound(specificSound, {
+                    position: _this.homePos,
+                    volume: AUDIO_VOLUME_LEVEL
+                });
+                playing = true;
+                Script.setTimeout(function() {
+                    playing = false;
+                }, 10);
```
Comment (misslivirose) [inline Marketplace/drumKit/ride.js]: Name variables

File: Marketplace/drumKit/snare.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+    var AUDIO_VOLUME_LEVEL = 1;
+    var playing = false;
+    var sound;
+
+    var Cowbell = function() {
+        _this = this;
+    };
+
+    Cowbell.prototype = {
```
Comment (misslivirose) [inline Marketplace/drumKit/snare.js]: still not a cowbell xD

File: Marketplace/drumKit/ride.js (hunk 4 above)
Comment (ZappoMan) [reply Marketplace/drumKit/ride.js]: It always needs more cowbell! 👍

**Comment by misslivirose** [comment]
Up on marketplace, passed marketplace QA

**Pull request merged**